core/src/main/java/tc/oc/pgm/PGMConfig.java

```diff
@@ -164,6 +165,8 @@
     this.verboseStats = parseBoolean(config.getString("ui.verbose-stats", "false"));
     this.griefScore =
         parseInteger(config.getString("gameplay.grief-score", "-10"), Range.atMost(0));
+    this.deathAnimationLength =
+        parseInteger(config.getString("gameplay.death-animation-length", "15"));
```

Comment:
`gameplay.death-ticks`

Reply:
Oh, also make sure it is `Range.atLeast(0)`. See the `griefScore` above for an example.